CSS/app.css

```diff
@@ -0,0 +1,287 @@
+
+.red {color: red;}
+
+.white{color: white;}
+
+.light {color: #6d6b6b;}
+
+.blue {color: blue;}
+
+.black {color: #000;}
+
+.yellow {color: yellow;}
+
+.maroon {color: #cf0000;}
+
+.bg-red {color: red;}
+.bg-white {color: white;}
+
+.bg-light {color: #cac7c7;}
+
+.bg-blue {color: blue;}
+
+.bg-black {color: #000000;}
+
+.bg-yellow {color: yellow;}
+
+.mgl {margin-left: 3rem;}
+
+.mgr {margin-right: 3rem;}
+
+.mgt {margin-top: 3rem;}
+
+.mgb {margin-bottom: 3rem}
+
+.family-roboto {font-family: "Roboto Condensed", sans-serif;}
+
+.family-Arial {font-family: Arial, Helvetica, sans-serif;}
+
+.family-helvetica {font-family: Helvetica, sans-serif;}
+
+.fs-1 {font-size: 1rem;}
+
+.fs-13 {font-size: 1.3rem;}
+
+.fs-14 {font-size: 1.4rem;}
+
+.fs-17 {font-size: 1.7rem;}
+
+.fs-20 {font-size: 2rem;}
+
+.border {border: 0.1rem solid black;}
+
+.ld-none {display: none;}
+
+html{
+  margin: 0;
+  padding: 0;
+  color: #000000;
+  font-size: 10px;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: 500;
+  box-sizing: border-box;
+  }
```

Comment:
### `HINT`

:bulb: Let's fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many IDE  can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

![Screenshot of how to format code using VS Code](https://res.cloudinary.com/bolaah/image/upload/v1563045186/github-microverse-project/code-reviews/code-review002.gif)